examples/kubernetes/nfs-provisioner/README.md

```diff
@@ -0,0 +1,18 @@
+# Set up a NFS Server on a Kubernetes cluster
+
+> Note: This example is for development perspective only. Because the NFS server is sticky to the node it is scheduled on, data shall be lost if the pod is rescheduled on another node.
+
```

Comment:
Thanks @msau42 , I have made the edit here mentioning that the example is for development only